### ISSUE TITLE
Help users get things in line for DUR

### ISSUE THREAD

**Issue by oneilsh** [National-COVID-Cohort-Collaborative/operations#99]
Since some of the pre-reqs for filling out the DUR should be completed before starting, should we warn or otherwise help users get those in place before navigating them to the DUR? (Idea: 3 or 4 checkboxes that could be clicked to make the DUR link clickable).

**Comment by oneilsh** [comment]
Duplicate of #89
